prototype_source/flight_recorder_tutorial.rst

```diff
@@ -48,6 +48,8 @@ Enabling Flight Recorder
 ------------------------
 There are two required environment variables to get the initial version of Flight Recorder working.
 
+- ``TORCH_NCCL_DEBUG_INFO_TEMP_FILE``: Setting the path where the flight recorder will be dumped with file prefix. One file per
```

Comment:
Nit: Can you move this to the **Optional settings** section?
Otherwise the comment on Line 49 can be confusing to the customer (there are two required environment variables).
Or you can fix the comment.